client/src/pages/UpdateTrainee.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import { Button } from "../component/Button";
+import "../pages/addTrainee.css";
+export const UpdateTrainee = () => {
+	const [fullname, setFullName] = useState("");
+	const [githubUserName, setGithubUserName] = useState("");
+	const [cohortId, setCohortId] = useState("");
+	const { id } = useParams();
+	console.log(id);
+	const navigate = useNavigate();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const response = await fetch(`/api/trainees/${id}`, {
+				method: "PUT",
+				headers: {
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify({
+					github_user_name: githubUserName,
+					full_name: fullname,
+					cohort_id: cohortId,
+				}),
+			});
+
+			if (!response.ok) {
+				throw new Error("couldn't to update trainee");
+			}
+
+			navigate(-1);
+		} catch (error) {
+			console.error("Error updating trainee:", error);
+		}
+	};
+
+	return (
+		<div className="addTrainee">
+			<h2>Update Trainee's details</h2>
+			<form onSubmit={handleSubmit}>
+				<label htmlFor="githubUsername">Github User Name </label>
+				<input
+					type="text"
+					placeholder="Github User Name"
+					value={githubUserName}
+					onChange={(e) => setGithubUserName(e.target.value)}
+				/>
```

Comment:
Please use Input component, you need to pass an id for the label to be associated with the input and placeholder shouldnt be the same as the label if used it should be an example of what would go in the field
```suggestion
				<Input
				  label="Github User Name"
					id="github-user-name"
					value={githubUserName}
					onChange={(e) => setGithubUserName(e.target.value)}
				/>
```